### PR TITLE
feat: support "create" on existing projects [sc-15038]

### PR DESCRIPTION
**Pull request by maxigimenez**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [ ] CLI
* [x] Create CLI
* [ ] Test
* [ ] Docs
* [ ] Examples
* [ ] Other

## Notes for the Reviewer
Re-opening #640 after a bug was found prior to release. The folder where dependencies were installed was wrong, now I'm sending `targetDir` correctly.

Here's the fix -> https://github.com/checkly/checkly-cli/pull/675/commits/746e2926f979b5857a6a4d4985d7edc8a85255f4

### CODE REVIEW

**Comment by shiini2** [comment]
How to run the CLI locally // against our BE/FE local env?

**Comment by ndom91** [comment]
> How to run the CLI locally // against our BE/FE local env?

Looks like you can pass a `CHECKLY_BASE_URL` environment variable (see `packages/cli/e2e/config/default.js:8`)

File: packages/create-cli/src/actions/dependencies.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { hint } from '../utils/messages.js'
+import { PackageJson } from '../utils/package.js'
+
+export function addDevDependecies (packageJson: PackageJson) {
+  if (!Reflect.has(packageJson, 'devDependencies')) {
+    packageJson.devDependencies = {}
+  }
+
+  Object.assign(packageJson.devDependencies, {
+    '@checkly/cli': 'latest',
```
Comment (ndom91) [inline packages/create-cli/src/actions/dependencies.ts:15]: This should also be `checkly` now :+1:

**Review by ndom91 — APPROVED** [review]
Other than the small pkg name thing, this LGTM. Works locally :+1:

File: packages/create-cli/src/actions/dependencies.ts (hunk 1 above)
Comment (maxigimenez) [reply packages/create-cli/src/actions/dependencies.ts]: Good catch!

**Review by ndom91 — APPROVED** [review]

**Comment by clample** [comment]
@maxigimenez should we do a new release of `create-checkly` or wait for more changes there?

For the release, we have it automated now with a GH action: https://github.com/checkly/checkly-cli/blob/main/CONTRIBUTING.md#releasing-create-checkly

**Pull request merged**